### PR TITLE
Fix textarea escaping

### PR DESCRIPTION
**Pull request by caveman99**
Address is a textarea field and needs to preserve linebreaks and whitespace

### CODE REVIEW

**Comment by caveman99** [comment]
Is this ever going to be merged? This is more than an annoyance.

**Comment by caveman99** [comment]
Hello? Anybody there? This is still an issue after the latest release.

**Review by imnavanath — APPROVED** [review]

**Review by premanshup — APPROVED** [review]

File: classes/widgets/class-astra-widget-address.php [hunk 1]
```diff
@@ -225,6 +225,9 @@ public function update( $new_instance, $old_instance ) {
 			$instance['display-icons'] = isset( $new_instance['display-icons'] ) ? (bool) $new_instance['display-icons'] : false;
 
 			$instance = array_map( 'sanitize_text_field', $instance );
+			
+			// Address is a textarea field and needs to preserve linebreaks and whitespace
+                        $instance['address'] = sanitize_textarea_field($new_instance['address']);
```
Comment (premanshup) [inline classes/widgets/class-astra-widget-address.php:230]:
```suggestion
			
			// Address is a textarea field and needs to preserve linebreaks and whitespace
                        if ( ! empty ( $new_instance['address'] ) ) {
				$instance['address'] = sanitize_textarea_field( $new_instance['address'] );
			}
```

**Comment by premanshup** [comment]
@caveman99 Thank you for the PR! Can you please make this one change? We will line this up for the release as soon as possible.

**Pull request merged**